.github/workflows/litex_sim.yml

```diff
@@ -76,24 +76,20 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: lschuermann/tock-litex
-          # Currently pointing to a fixed commit, switch to a tag on the next
-          # release with the Python requirements.txt included.
-          #
-          # Also, currently using the dev/litex-sim-ci branch, which
-          # pulls in a forked LiteX with GPIO support in the
-          # simulation and the simctrl modules
-          ref: da3f33968ae018c33dd10d705fd7e3ca04e5e056
+          # The pinned revision is different from the targeted release as
+          # documented in the LiteX boards, as the CI requires special patches
+          # to LiteX for interacting with the simulation:
+          ref: tock-cipin-1
           path: tock-litex
 
       # Install all of the required Python packages from the tock-litex'
       # requirements.txt file
       - name: Install Python packages pinned by the tock-litex revision
         run: |
           pushd tock-litex
-          # Unfortunately the valentyusb package fails to install because
-          # it does not have a setup.py. Thus ignore it, we don't need it
-          # here anyways (would need it for LiteX unit tests though).
-          sed -i '/valentyusb/d' ./requirements.txt
+          # Migen is technically not a LiteX package, which is why it is not
+          # contained within the requirements.txt file:
```

Comment:
what is it then / why do we need it (when litex doesn't?)

Reply:
Migen is the DSL which the LiteX ecosystem uses as its hardware-description language. It effectively provides a set of Python classes and constructs which can be translated into Verilog. So it's not a package of the LiteX ecosystem, but it is a hard requirement. I don't include it in the auto-generated requirements.txt of `tock-litex` given it's a regular system dependency and we don't hard-require a specific or patched version of it.

Reply:
```suggestion
          # Migen is the DSL which the LiteX ecosystem uses as its
          # hardware-description language. It effectively provides a set of
          # Python classes and constructs which can be translated into Verilog.
          # It is not a package of the LiteX ecosystem, and thus not in the
          # requirements.txt, but it is required to be present on the system.
          # It should not requie any specific or patched version.
```

Reply:
Fair enough :)